packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -612,20 +613,23 @@ initialize ()
       bool use_explicit_conversion = List_.isParameter("partitioner: explicit convert to BlockCrs") && List_.get<bool>("partitioner: explicit convert to BlockCrs");
       TEUCHOS_TEST_FOR_EXCEPT_MSG
         (use_explicit_conversion && block_size == -1, "A pointwise matrix and block_size = -1 were given as inputs.");
+      bool use_LID = !List_.isParameter("partitioner: use LIDs") || List_.get<bool>("partitioner: use LIDs");
```

Comment:
Is this what you intend?

Reply:
Yes, by default I wanted to use LIDs as it is faster. What do you think of that?

Reply:
Oh, that makes sense.  The first argument of the `||` statement was throwing me.